vite-app/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
```

Comment:
I added a vite app jsut to confirm that the issues in the other app were definitely related to ssr, and that was confirmed. `lib1` and `lib2` work as expected in vite with no changes as compared to `main`.